### ISSUE TITLE
Nothing Showing up

### ISSUE THREAD

**Issue by sstar0016** [LynnXie00/ObsidianGanntChartfromTasks#1]
Hi,
I really like the idea of your dataview code but unfortunately I have not been able to get it to work. I have changed the directory to the top level folder and still nothing is showing up. Wondering if you had any tips?
![image](https://user-images.githubusercontent.com/124852454/218811990-581b9634-70f5-49d1-ba97-79297ebd82ac.png)
![image](https://user-images.githubusercontent.com/124852454/218812461-d788eb0a-23bb-403a-b665-b2fa9e5d49f0.png)

![image](https://user-images.githubusercontent.com/124852454/218812267-d1b3652c-442b-4d40-8a68-b6e6cf997f77.png)

**Comment by LynnXie00** [comment]
> Hi, I really like the idea of your dataview code but unfortunately I have not been able to get it to work. I have changed the directory to the top level folder and still nothing is showing up. Wondering if you had any tips?
> 
>

Do you use the plugin obsidian tasks? My code is written for those who use obsidian tasks.

https://github.com/obsidian-tasks-group/obsidian-tasks

**Comment by dxcore35** [comment]
Please make it generic. Just simple tasks formatting also 🙏🏻:
```
- [ ] YYYY-MM-DD
- [x] YYYY-MM-DD
```

**Comment by sstar0016** [comment]
> > Hi, I really like the idea of your dataview code but unfortunately I have not been able to get it to work. I have changed the directory to the top level folder and still nothing is showing up. Wondering if you had any tips?
> 
> Do you use the plugin obsidian tasks? My code is written for those who use obsidian tasks.
> 
> https://github.com/obsidian-tasks-group/obsidian-tasks

Yes I am using it but as can be seen from my screenshot above it doesnt seem to work. Am I missing a step to enable anything?
